README.md

```diff
@@ -6,16 +6,31 @@ Searchfox locally using Vagrant.
 
 ## Vagrant setup for local development
 
-First, [install Vagrant](https://www.vagrantup.com/downloads.html) and
-[VirtualBox](https://www.virtualbox.org/wiki/Downloads) by
-following the instructions for your OS. Then clone Mozsearch and
-provision a Vagrant instance:
-
+First, [install Vagrant](https://www.vagrantup.com/downloads.html).  This is a
+virtualization automation tool.  It depends on a virtualization backend.
+
+Pick a virtualization backend and install it.  Your options are:
+- libvirt via [vagrant-libvirt](https://github.com/vagrant-libvirt/vagrant-libvirt).
+  If you are on linux and aren't already using VirtualBox, it's the recommended
+  path.
+  - Follow the [installation instructions](https://github.com/vagrant-libvirt/vagrant-libvirt#installation).
+  - Note that if you didn't already have libvirt installed, then a new `libvirt`
+    group may just have been created and your existing logins won't have the
+    permissions necessary to talk to the management socket.  If you do
+    `exec su -l $USER` you can get access to your newly assigned group.
```

Comment:
Could you add a section like this to the docs? I had this locally, but it seems I never pushed it:

> ## Troubleshooting
> 
> If vagrant up times out in the "Mounting NFS shared folders..." step, chances
> are that you cannot access nfs from the virtual machine.
> 
> You probably need to allow `libvirt` to access `nfs`:
> 
> ```
> # firewall-cmd --permanent --add-service=nfs --zone=libvirt
> # firewall-cmd --permanent --add-service=rpc-bind --zone=libvirt
> # firewall-cmd --permanent --add-service=mountd --zone=libvirt
> # firewall-cmd --reload
> ```

I had to do this for a new Fedora 31 machine.

Reply:
Okay, I added this.